CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.22.1 FATAL_ERROR)
```

Comment:
I don't think it's a good idea to change this in a stable release. We're making this change in `main`.

Reply:
arg, sorry, lost track while reviewing Steve gz-math's changes. I leave all the code as it was before.